app/helpers/application_helper.rb

```diff
@@ -97,15 +97,6 @@ def render_map(lat, lon)
     render partial: 'map/leaflet', locals: { lat: lat, lon: lon }
   end
 
-  # we should move this to the Comment model:
-  # returns the comment body which is to be shown in the comments section
-  def render_comment_body(comment)
-    raw RDiscount.new(
-      title_suggestion(comment),
-      :autolink
-    ).to_html
-  end
-
   # we should move this to the Comment model:
   # replaces inline title suggestion(e.g: {New Title}) with the required link to change the title
   def title_suggestion(comment)
```

Comment:
I'm only a little worried that we will still need this filter, and not sure if it's available inside the comment.rb file. Would you mind starting up the app and seeing on any note page (say, click any post shown on http://localhost:3000/research) to ensure that it's getting properly displayed? 

I /think/ tests ought to trip if this did go wrong, but a screenshot of a note page would be comforting!

Thank you!!

Reply:
Another possibility is that if comments DO have trouble, we could ALSO move the `title_suggestion` method into comment.rb:

https://github.com/publiclab/plots2/blob/ee5a7b120cc07bf678df480c6f404cc96ee09d04/app/helpers/application_helper.rb#L100-L117

What do you think?

Reply:
I was actually thinking about moving title_suggestions too, but I thought it would be better to stay in the context of the issue, as I don't know much about the codebase yet. I'll startup the app in a sec and report back.

Reply:
I had to remove the ```raw``` statement from ```def render_body```other than that it seems to work, also attached a screenshot. A commit will follow shortly.

![screenshot 2018-10-03 at 01 34 36](https://user-images.githubusercontent.com/17762773/46382701-ad5c9300-c6ac-11e8-8579-3d62e257842b.png)

If you would like me to still move ```def title_suggestion```I can do that.

As a side note ```find . -name '*.rb' | xargs grep 'render_comment_body'```returned 0 results, so it isn't referenced anywhere else.

Reply:
Now that i think of it, I believe we need the `raw` segment for some JavaScript work we do to scrape the original markdown. Do you think you could add it back in? 

Good to hear about the grep. Thanks!

Reply:
@jywarren ```raw``` isn't available from a model, only from a controller.

Reply:
No problem, embedded HTML works (```<a class="btn btn-primary">Hello</a>```). Not sure about a youtube video, as I don't know how one does embed it, but I think that the ```raw```statement probably was redundant.